stubs/init-laravel/phpunit.xml.stub

```diff
@@ -4,28 +4,29 @@
          bootstrap="vendor/autoload.php"
          colors="true"
 >
-    <testsuites>
-        <testsuite name="Unit">
-            <directory suffix="Test.php">./tests/Unit</directory>
-        </testsuite>
-        <testsuite name="Feature">
-            <directory suffix="Test.php">./tests/Feature</directory>
-        </testsuite>
-    </testsuites>
-    <coverage>
-        <include>
-            <directory suffix=".php">./app</directory>
-        </include>
-    </coverage>
-    <php>
-        <env name="APP_ENV" value="testing"/>
-        <env name="BCRYPT_ROUNDS" value="4"/>
-        <env name="CACHE_DRIVER" value="array"/>
-        <!-- <env name="DB_CONNECTION" value="sqlite"/> -->
-        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
-        <env name="MAIL_MAILER" value="array"/>
-        <env name="QUEUE_CONNECTION" value="sync"/>
-        <env name="SESSION_DRIVER" value="array"/>
-        <env name="TELESCOPE_ENABLED" value="false"/>
-    </php>
+  	<testsuites>
+    	<testsuite name="Unit">
+      		<directory suffix="Test.php">./tests/Unit</directory>
+    	</testsuite>
+    	<testsuite name="Feature">
+      		<directory suffix="Test.php">./tests/Feature</directory>
+    	</testsuite>
+  	</testsuites>
+  	<coverage/>
+  	<php>
+    	<env name="APP_ENV" value="testing"/>
+    	<env name="BCRYPT_ROUNDS" value="4"/>
+    	<env name="CACHE_DRIVER" value="array"/>
+    	<!-- <env name="DB_CONNECTION" value="sqlite"/> -->
+    	<!-- <env name="DB_DATABASE" value=":memory:"/> -->
+    	<env name="MAIL_MAILER" value="array"/>
+    	<env name="QUEUE_CONNECTION" value="sync"/>
+    	<env name="SESSION_DRIVER" value="array"/>
+    	<env name="TELESCOPE_ENABLED" value="false"/>
+  	</php>
+  	<source>
+    	<include>
+      		<directory suffix=".php">./app</directory>
+    	</include>
```

Comment:
What's this coverage?

Reply:
Do I answer to your question in the conversation above ? Maybe am I missing something.